tests/expect/test_fish_completion.exp

```diff
@@ -35,7 +29,7 @@ puts "\nMatching:\n"
 foreach arg [lrange $argv 3 end] {
     send "\t"
     expect {
-	"$arg" {puts "matched '$arg' to '$expect_out(0,string)'\nbuffer: '$expect_out(buffer)'";continue}
+	-re "$arg" {puts "matched '$arg' to '$expect_out(0,string)'\nbuffer: '$expect_out(buffer)'";continue}
```

Comment:
I can
can't quiet remember what -re is doing.
please explain.

Reply:
Matching regexp. "$arg" would be regexp as we want to do exact line end match.

Reply:
are you escaping the inputs?
expected strings have ., you should re.escape() them before passing them otherwise the . would interact with the regex matching.

Reply:
yes they are escaped.